internal/grpc/services/storageprovider/storageprovider.go

```diff
@@ -709,6 +709,20 @@ func (s *service) Delete(ctx context.Context, req *provider.DeleteRequest) (*pro
 		}, nil
 	}
 
+	if utils.ReadPlainFromOpaque(md.GetOpaque(), "status") == "processing" {
```

Comment:
The GetMD() call in line 705 only requests the `id`. Please add `status` so that if we implement the filter we can actually see that `session` is required.

Reply:
The "status" is included in GetMD() implicitly.  Is it by design?

Reply:
https://github.com/cs3org/reva/blob/edge/pkg/storage/utils/decomposedfs/node/node.go#L648C12-L648C12

Reply:
The "status" is included if n.IsProcessing(ctx)